server/lib/user-aws-credential-generator.js

```diff
@@ -95,6 +95,20 @@ class UserAwsCredentialGenerator {
           "Effect": "Allow",
           "Action": "s3:DeleteObject",
           "Resource": "${this.arnS3Prefix()}/*"
+        },
+        {
```

Comment:
does this work? i'm curious if the sync server AWS IAM profile needs to be updated to have to permission to do this as well. iirc the child credentials can only authorize a subset of the parent permissions.

Reply:
I haven't checked the Rest Sync, but I see that I haven't implemented the delete of SNS and SQS. I'll fix that tomorrow.